addons/project/models/project_project.py

```diff
@@ -456,6 +458,9 @@ def copy(self, default=None):
             project.milestone_ids = [milestone.copy().id for milestone in self.milestone_ids]
         if 'tasks' not in default:
             self.map_tasks(project.id)
+        if not self.active:
+            tasks = self.env['project.task'].with_context(active_test=False).search([('project_id', '=', project.id)])
+            tasks.write({'active': True})
```

Comment:
```suggestion
        if not self.active:
            project.with_context(active_test=False).tasks.active = True
```
Doesn't that accomplish the same thing?